packages/now-static-build/test/fixtures/52-ionic-angular/package.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
This test fixture is failing with `Error: ENOENT: no such file or directory, scandir '/zeit/416b9c19/dist'`